doc/internalconsole.rst

```diff
@@ -1,17 +1,34 @@
+################
 Internal Console
-================
+################
 
-The **Internal Console** is dedicated to Spyder internal debugging or may be
-used as an embedded Python console in your own application.
-All the commands entered in the internal console are executed in the same
-process as Spyder's, but the Internal Console may be executed in a separate
-thread (this is optional and for example this is not the case in Spyder itself).
+You can use the **Internal Console** to control and debug Spyder itself, or as an embedded Python console in your own application.
 
 .. image:: images/internal_console/internal_console_standard.png
    :alt: Spyder Internal Console, with error messages and command output shown
 
 |
 
-The internal console support the following features:
 
-* Code completion and calltips
+==================
+Supported features
+==================
+
+The :guilabel:`Internal Console` supports code completion and calltips, just like a normal :doc:`ipythonconsole`, along with a number of GUI selection, line-wrapping, buffer, and clear/copy/paste options.
+You can toggle and customize these behaviors under :guilabel:`Internal console settings` from the context menu.
+Additionally, you can use a number of special commands can be used to open a file in a Spyder or external editor, edit or remove variables, run scripts or system commands, get help, and more.
```

Comment:
Either `you can use` or `can be used` needs to be removed here.

Reply:
Yup, thanks for the catch. Fixed.